### DISCUSSION TITLE
January Requirements Question

### DISCUSSION THREAD

**Discussion by vduffel99** [CMSgov/hospital-price-transparency#113] [Q&A] [answered]
Hi All - I have a question about the January requirements, more specifically the estimated amounts. My annual files get completed around July of each year. I am including the January requirements in my file but wanted to know if there are changes to the requirements between now and January 1, will I need to update the file to comply with the updated changes? See below for examples.

1. Change to what is allowed to be leveraged for estimated amounts (requiring 835s to be used)
2. Changes to how we list out modifiers in the file

As always thank you for taking the time to help me better understand the requirements.

**Comment by RidenourK** [comment, marked as answer]
@vduffel99. Thank you for your questions. 

In the CY 2024 OPPS/ASC Final Rule, we indicated we believe hospitals should retain flexibility, in the interest of reducing burden, to determine the best data source for calculating the estimated allowed amount. We therefore declined at the time to be prescriptive. However, we indicated we agree that using information from the EDI 835 electronic remittance advice (ERA) transaction, the electronic transaction that provides claim payment information, including any adjustments made to the claim, such as denials, reductions, or increases in payment, would appear to meet this requirement as the data in the 835 form is used by hospitals to track and analyze their claims and reimbursement patterns. Please refer to 88 FR 82100.

Please refer to the CSV and JSON documentation, templates and examples on the Hospital Price Transparency – Data Dictionary GitHub for information and examples on how to encode the estimated allowed amount and modifiers.
